### PR TITLE
Documentation Added for IS-5.9.0 Improvement to Account Lock/Unlock Event and Email Template 5.9 Fix

### PR DESCRIPTION
**Pull request by Buddhimah**
https://github.com/wso2/product-is/issues/6778

### CODE REVIEW

File: en/docs/learn/account-locking-by-failed-login-attempts.md [hunk 1]
```diff
@@ -210,4 +210,137 @@ failed login attempts. To configure this, follow the steps below.
...
+!!! warning
+    Make sure you have a WUM updated WSO2 Identity Server 5.9.0 pack. If not this section is not relevant.
+    For more information on how to WUM update, see [Updating WSO2 Products](../../administer/getting-wso2-updates)
     
+WUM Updated WSO2 Identity Server 5.9.0 requires separate email templates for notifying, 
+
+- Account locking by exceeding `Maximum Failed Login Attempts`
+- Account unlocking by exceeding `Account Unlock Time`
+
+Please add the following email templates by referring to the instructions in
```
Comment (sherenewso2) [inline en/docs/learn/account-locking-by-failed-login-attempts.md]: Generally we avoid using the word "please" in our docs to keep the tone neutral. We can start this sentence with "Add the following..."

File: en/docs/learn/account-locking-by-failed-login-attempts.md [hunk 2]
```diff
@@ -210,4 +210,137 @@ failed login attempts. To configure this, follow the steps below.
         can edit and customize the email template. For more information on
         how to do this, see [Customizing Automated
         Emails](../../learn/customizing-automated-emails).
+
+#### Configuring email templates for WUM Updated version
+
+!!! warning
+    Make sure you have a WUM updated WSO2 Identity Server 5.9.0 pack. If not this section is not relevant.
+    For more information on how to WUM update, see [Updating WSO2 Products](../../administer/getting-wso2-updates)
```
[the remark this answers is not in the document]
Comment (Buddhimah) [reply en/docs/learn/account-locking-by-failed-login-attempts.md]: done

File: en/docs/learn/account-locking-by-failed-login-attempts.md [hunk 3]
```diff
@@ -210,4 +210,137 @@ failed login attempts. To configure this, follow the steps below.
...
     
+WUM Updated WSO2 Identity Server 5.9.0 requires separate email templates for notifying,
```
[the remark this answers is not in the document]
Comment (Buddhimah) [reply en/docs/learn/account-locking-by-failed-login-attempts.md]: done

File: en/docs/learn/account-locking-by-failed-login-attempts.md (hunk 1 above)
Comment (Buddhimah) [reply en/docs/learn/account-locking-by-failed-login-attempts.md]: done

**Pull request merged**